src/zoid/button/component.jsx

```diff
@@ -230,6 +232,15 @@ export const getAuthButtonComponent = memoize(() : ZoidComponent<ButtonProps> =>
                 }
             },
 
+            inputLabel: {
```

Comment:
Why do we need a new prop? Can we not continue to use `style.label`?

Reply:
@bluepnume this is a input label type where we get the value from the integration partner, to specify what label they want to render in the button.

Reply:
I don't see this in the spec PR: https://github.com/paypal/paypal-sdk-spec/pull/55 -- seems like we should remove it here, right? Given that we already have `style.label`

Reply:
yes